openmc/lattice.py

```diff
@@ -1059,15 +1139,27 @@ def find_element(self, point):
         else:
             z = point[2] - self.center[2]
             iz = floor(z/self.pitch[1] + 0.5*self.num_axial)
-        alpha = y - x/sqrt(3.)
-        ix = floor(x/(sqrt(0.75) * self.pitch[0]))
-        ia = floor(alpha/self.pitch[0])
+        if self._hextype:#//DR
+           alpha = y - x*sqrt(3.)
+           ix = 0
+           ia = floor(-alpha/(sqrt(3.0) * self.pitch[0]))
+           iy = floor(y/(sqrt(0.75) * self.pitch[0]))
+        else:
+           alpha = y - x/sqrt(3.)
+           ix = floor(x/(sqrt(0.75) * self.pitch[0]))
+           ia = floor(alpha/self.pitch[0])
+           iy = 0
```

Comment:
It would be better to use `i1, i2` instead of `ix/ia, ia/iy` like you did in `HexLattice::get_indices` in lattice.cpp.  That will reduce the amount of code needed here, and it's always better for the Python code to match the C++ code as much as possible.

Indent 4 spaces rather than 3.